gbfs.md

```diff
@@ -96,13 +107,16 @@ Field Name          | Required  | Defines
 --------------------| ----------| ----------
 last_updated        | Yes       | Integer POSIX timestamp indicating the last time the data in this feed was updated
 ttl                 | Yes       | Integer representing the number of seconds before the data in this feed will be updated again (0 if the data should always be refreshed)
+version             | Yes       | GBFS version number in which feed data is encoded, according to the versioning framework.
```

Comment:
`GBFS version number in which feed data is encoded` -> GBFS version number to which the feed conforms`

Reply:
It would be good to indicate the data type (string) here as well.

Reply:
I made the suggested changes.